frontend/package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "frontend",
```

Comment:
Would it be possible to call it `credit-application-frontend` to make in consistent with the documentation?

Reply:
done